PluginGenerator/JarBuilder.cs

```diff
@@ -170,7 +170,7 @@ private void WriteManifestFile(string manifestFilePath)
             StringBuilder sb = new StringBuilder();
             foreach (KeyValuePair<string, string> kvp in this.manifestProperties)
             {
-                sb.AppendFormat(System.Globalization.CultureInfo.CurrentCulture,
```

Comment:
We don't want culture-sensitive behaviour when building the key-value pair strings, or when building the command line arguments below.